chicken-sloths/bangazon-site-sprint3
# /cart displays appropriate message for deleting products from cart

# Description
`/cart` displays products. If there aren't any in it, then a 'complete order' no longer appears. Instead, a 'product categories' button appears and a message appears telling the customer to put some shit in that cart!

## Related Ticket(s)
Fixes #76 

## Expected Behavior
Deleting everything from the cart should change the button to 'product categories' and direct to `/` when clicked.

## Steps to Test Solution

1. `npm start`
1. Sign in with a user (my personal fav: `Aletha.Labadie34@gmail.com`)
1. Go to their cart. It should display the appropriate thing
1. Delete everything from the cart if there's anything in there
1. Click on the new 'product categories' btn
1. Add stuff to the cart and delete it all again
1. Magic!
